broken-line/contracts/LibBrokenLine.sol

```diff
@@ -51,8 +65,52 @@ library LibBrokenLine {
     }
 
     /**
-     * Обновляет initial линию для переданного time. Высчитывает и применяет все изменения из slopeChanges за этот период
-    **/
+     *remove Line from BrokenLine, return line.bias, which actual now moment
+     **/
+    function remove(BrokenLineDomain.BrokenLine storage brokenLine, uint id, uint toTime) internal returns (uint) {
+        BrokenLineDomain.LineData memory lineData = brokenLine.initiatedLines[id];
+        BrokenLineDomain.Line memory line = lineData.line;
+        require(line.bias != 0, "Line with given id already finished");
+
+        update(brokenLine, toTime);
+        //check time Line is over
+        uint period = line.bias.div(line.slope);
+        uint finishTime = line.start.add(period).add(lineData.cliff);
+        if (toTime > finishTime) {
+            line.bias = 0;
+            return 0;
+        }
+        uint finishTimeMinusOne = finishTime.sub(1);
+        int mod = safeInt(line.bias.mod(line.slope));
+        uint nowBias = line.bias;
```

Comment:
line.bias is used before this line. is it better to move this variable declaration to the start of the function?
Also, bias is better naming than nowBias I think
Also, why is it named nowBias?

Reply:
nowBias - bias in now moment, I agree name should contain meaning main goal of variable.
agree bias, should work from the start.